nss-day-cohort-48/the-worst-site-ever-eliminatorz
# Adding madeline.html and madeline.css

Adding the first draft of madeline.html and madeline.css

##steps to test

1. Pull this branch
2. madeline.html should be in the pages folder and madeline.css should be in the styles folder.
3. a link to Madeline's page should display in index.html
4. open madeline.html in chrome
5. "hello!" header and 3 pics should display
6. navbar should display with home going back to my page and the other teammates' links going to the wrong teammates
